base/strings/basic.jl

```diff
@@ -630,6 +653,36 @@ end
     end
     return  _isascii(cu,last-chunk_size+1,last)
 end
+
+function isascii(bytes::AbstractVector{UInt8}, first, last)
+    q_chunk_size = 128
+
+    n = last - first + 1
+    nqwords, epilog_bytes = (n >> 3, n & 0x07) # divrem(n,8)
+    epilog_first = last - epilog_bytes + 1
+    if epilog_bytes > 0
+        _isascii_subqword(bytes,epilog_first,epilog_bytes) || return false
+    end
+    nqwords > 0 || return true
+    nchunks, prolog_qwords = divrem(nqwords,q_chunk_size)
+
+    qwords = @inline @inbounds reinterpret(UInt64,@inbounds view(bytes,first:(epilog_first - 1)))
+
+    qwords_start = 1
+    if prolog_qwords > 0
+        _isascii_qword(qwords,qwords_start,prolog_qwords) || return false
```

Comment:
```suggestion
        _isascii_qword(qwords, qwords_start, prolog_qwords) || return false
```

It's not explicitly mentioned in [CONTRIBUTING.md](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-julia-code-contributions), there's just a  generic "use whitespace for better readability", but I think putting some whitespace around arguments makes it a bit easier on the eyes, especially since all arguments contain `qwords`.